tools/devserver/routers/sample.js

```diff
@@ -100,4 +100,26 @@ module.exports = function(server){
 		setTimeout_2();
 
 	});
+
+	//e. sample JSON Web Token login
+	//user provide username and password, and server checks whether username and password is valid or not.
+	//if yes, return token, encrypted by server.jwt. if not return 403.
+	router.post('/jwt/login', function(req, res, next){
```

Comment:
We need to add another sample route (e.g /jwt/check) to demo the usage and checking of token's permission of the user included in the payload (now should be in req.token.permissions)
Having said that, we might need to change how the permissions are checked in the router in my devserver implementation.